sdk/README.md

```diff
@@ -63,7 +63,7 @@ The `code-push` module exports a single class (typically referred to as `CodePus
 
 - __getDeploymentHistory(appName: string, deploymentName: string): Promise&lt;Package[]&gt;__ - Retrieves the list of releases that have been made to the specified app deployment.
 
-- __getDeploymentMetrics(appName: string, deploymentName): Promise&lt;DeploymentMetrics&gt;__ - Retrieves the installation metrics for the specified app deployment. 
+- __getDeploymentMetrics(appName: string, deploymentName:string): Promise&lt;DeploymentMetrics&gt;__ - Retrieves the installation metrics for the specified app deployment.
```

Comment:
Could you please add whitespace before `string` word for keeping code style? 
It should be like so: `deploymentName: string`

Reply:
Yeah, I'll do it and resubmit my PR. Thanks.